pensjonsmaler/src/main/kotlin/no/nav/pensjon/brev/maler/fraser/alderspensjon/InnvilgelseFelles.kt

```diff
@@ -0,0 +1,46 @@
+package no.nav.pensjon.brev.maler.fraser.alderspensjon
+
+import no.nav.pensjon.brev.template.Expression
+import no.nav.pensjon.brev.template.LangBokmalNynorskEnglish
+import no.nav.pensjon.brev.template.Language.Bokmal
+import no.nav.pensjon.brev.template.Language.English
+import no.nav.pensjon.brev.template.Language.Nynorsk
+import no.nav.pensjon.brev.template.OutlinePhrase
+import no.nav.pensjon.brev.template.dsl.OutlineOnlyScope
+import no.nav.pensjon.brev.template.dsl.expression.expr
+import no.nav.pensjon.brev.template.dsl.expression.ifElse
+import no.nav.pensjon.brev.template.dsl.expression.or
+import no.nav.pensjon.brev.template.dsl.expression.plus
+import no.nav.pensjon.brev.template.dsl.textExpr
+
+// hvisFlyttetBosattEØS / hvisFlyttetBosattAvtaleland
+data class HvisFlytetFaktiskBostedsland(
```

Comment:
Lurt med frase for dette, kan jo fint ta den i bruk frå innvilgelse-manuelt og vedtak endring flytting mellom land med det samme.